nemo_curator/download/commoncrawl.py

```diff
@@ -288,7 +363,7 @@ def download_common_crawl(
     url_limit=None,
 ) -> DocumentDataset:
```

Comment:
From my understanding, the `download_common_crawl` function is what the user calls in their Python script. However, it doesn't look like they can set any of the `extract_text` parameters themselves (like `length_low`, `length_high`, etc.). Should we update this so that the user has more control over these parameters?